src/components/TodoList.tsx

```diff
@@ -0,0 +1,204 @@
+import {
+  ChangeEvent, FocusEvent, FormEvent,
+  KeyboardEvent, useEffect, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
+import { deleteTodo, updateTodo } from '../api/todos';
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[],
+  filter: Filter,
+  tempTodo: Todo | null,
+  handleError: (error: string) => void,
+  isUpdating: boolean,
+  handleIsUpdating: (status: boolean) => void,
+  updatingIds: number[],
+  handleUpdatingIds: (ids: number[]) => void,
+  handleLoadTodos: () => void,
+}
+
+export const TodoList: React.FC<Props> = ({
+  todos, filter, tempTodo, handleError, handleLoadTodos,
+  isUpdating, handleIsUpdating, updatingIds, handleUpdatingIds,
+}) => {
+  const [editTodo, setEditTodo] = useState<Todo | null>(null);
+  const [newTitle, setNewTitle] = useState('');
+
+  useEffect(() => {
+    handleLoadTodos();
+  }, [isUpdating]);
```

Comment:
Remove this.
You need to have only 1 `useEffect` with a fetch request on the initial render in this task
 
```suggestion
```